components/ILIAS/Dashboard/Administration/classes/class.ilDashboardSortationTableGUI.php

```diff
@@ -71,12 +73,13 @@ public function initData(): void
             $presentation_cb->setChecked($this->viewSettings->isViewEnabled($presentation_view));
             $presentation_cb->setValue('1');
             $presentation_cb->setDisabled(
-                $presentation_view === ilPDSelectedItemsBlockConstants::VIEW_RECOMMENDED_CONTENT
+                $disable ?? ($presentation_view === ilPDSelectedItemsBlockConstants::VIEW_RECOMMENDED_CONTENT)
```

Comment:
Do you mean `?:` instead of `??`?. If I see this correctly `$disable` cannot be nullable, as `ilNumberInputHUI::setDisabled` requires a non nullable boolean.
This means `$disable ?? 'foo'` will never result in `'foo'`, because `??` only "reacts" on `null`